cli/crates/server/src/servers.rs

```diff
@@ -70,8 +70,10 @@ pub fn start(port: u16, watch: bool, tracing: bool) -> (JoinHandle<Result<(), Se
 
                     tokio::select! {
                         result = start_watcher(environment.project_grafbase_path.clone(),  move |path, event_type| {
-                            let relative_path = path.strip_prefix(&environment.project_path).expect("must succeed by definition").to_owned();
-                            watch_event_bus.send(Event::Reload(relative_path, event_type)).expect("cannot fail");
+                            if !path.starts_with(&environment.user_dot_grafbase_path) {
```

Comment:
Why? Sure if the cache is updated we'll reload stuff, so we'll do more reloading than necessary, but we'll still reload properly if hot reload was expected. IMHO this is a place where specifying a different shared cache directory (home) would be useful, the new home could just be `~/.grafbase-dynamodb`.